bin/faucet/src/server/pow.rs

```diff
@@ -222,183 +249,94 @@ impl ChallengeCache {
 
 #[derive(Serialize)]
 struct PoWResponse {
-    seed: String,
+    challenge: String,
     difficulty: usize,
-    server_signature: String,
     timestamp: u64,
 }
```

Comment:
Why do we need this `PoWResponse` struct? Isn't everything we need already encoded in the `challenge` string?